ToolBox/Visualizer/QuantConnect.Visualizer.py

```diff
@@ -0,0 +1,182 @@
+"""
+Usage:
+    QuantConnect.Visualizer.py DATAFILE [--output file_path] [--size height,width] [--reset]
+
+Arguments:
+    DATAFILE   Absolute or relative path to a zipped data file to plot.
+               Optionally the zip entry file can be declared by using '#' as separator.
+
+Options:
+    -h --help                 show this.
+    -o --output file_path     path or filename for the output plot. If not declared, it will save with an
+                              auto-generated name at the default folder defined in the config.json file.
+    -s, --size height,width   plot size in pixels [default: 800,400].
+    -r, --reset               Resets it forces the writing of the config.json file in the assembly folder.
```

Comment:
I don't believe this is needed. Since we set the `config.json`, if should be always good. If not, we throw an exception and edit it.